src/main/java/io/supertokens/session/Session.java

```diff
@@ -56,15 +56,17 @@
 
     @TestOnly
     public static SessionInformationHolder createNewSession(Main main, @Nonnull String userId,
-            @Nonnull JsonObject userDataInJWT, @Nonnull JsonObject userDataInDatabase)
+            @Nonnull JsonObject userDataInJWT, @Nullable JsonObject grantPayload,
+            @Nonnull JsonObject userDataInDatabase)
```

Comment:
To ease the changes required for the tests, maybe we can create two versions of this function -> one which doesn't take `grantPayload`, and one that does. This way, the tests that have nothing to do with `grantPayload` don't need to change.

But since you have already made all the changes, I guess it's OK.